website/server/models/user/hooks.js

```diff
@@ -254,6 +258,10 @@ schema.pre('save', true, function preSaveUser (next, done) {
         this.notifications.push(lastExistingNotification);
       }
     }
+
+    notificationsToAdd.forEach(notification => {
```

Comment:
since we've been using it across the codebase could you change this to use `user.addNotification()` instead of pushing to the array ?

Reply:
Also no need to create a temporary array and then push them together at the end